### ISSUE TITLE
Flow-Router/Iron-Router mysteriously stops working when Famous-Views is added

### ISSUE THREAD

**Issue by ashah888** [gadicc/meteor-famous-views#258]
See title.

The router's action never gets called when Famous-Views is installed. Before the installation, it works fine.

**Comment by pmp0x** [comment]
I have the same problem. It seems to be that is has something to do how the new Tracker is working. I do not have time to investigate. But a workaround is to remove the tracker.js from the package file.

**Comment by gadicc** [comment]
Thanks guys.  Sorry for the lack of action until now.  This is super serious and will be treated appropriately.  I've been waiting for a few serious (unrelated) bugs to be resolved from the famous side, but I think I'm going to make some effort to work around them so we can get development back on track.  I'll post when I have some news.

In the mean time, if you're up for troubleshooting, you can see if placing famous-views at the top of your `.meteor/packages` file makes any difference, and if, when viewing source in devel, the router appears between Tracker and famous-views packages.

**Comment by gadicc** [comment]
I think I know exactly what's going on but would be super helpful if someone can create a cloneable repro that consistently shows this.  I've used the tracker code extensively with both iron-router and flow-router without ever running into this in v0, did you ever use that version as well?  I'm trying to think if I changed anything since then.

**Comment by PhoenixWright** [comment]
Hey @gadicc, I just tried placing famous-views at the top of .meteor/packages and I got my site back. I was incredibly confused until I finally found this issue... might want to mention this in the readme given how likely using one of these routers is. Just in case you might want this, here's my edited file:

meteor-platform
autopublish
insecure
accounts-password
jag:pince
coffeescript
mquandalle:jade
gadicohen:famous@=0.5.2
gadicohen:famous-views
kadira:flow-router
kadira:blaze-layout
accounts-ui
reactive-dict
reactive-var
benmgreene:dagre
mquandalle:stylus

**Comment by gadicc** [comment]
@PhoenixWright, well, that's more of a workaround... it shouldn't actually be necessary but it did help me understand where things were going wrong.  So with the original order and the latest version of famous-views, the bug still exists?

**Comment by PhoenixWright** [comment]
Yes, you can be reasonably assured these were all updated within a couple of days. I did make the famous version workaround suggested (the 0.5.2 one). It was a pretty easy repro - essentially I was adding and removing famous-views and having my site turn into one with a blank body whenever it was added. I just added you to a private repo for the sake of repro -  this is grotesquely non-public, but here's the commit where I added famous-views: https://github.com/PhoenixWright/adventure-tab/commit/c8145054318d0d12624d9fe267160240e9274d59

You should be able to grab the commit before that: https://github.com/PhoenixWright/adventure-tab/commit/fb009ea8fd70c41ce375411d2b94526a7eb7fe13 and try adding famous-views and seeing the site disappear after the addition.
